### DISCUSSION TITLE
device failed to set device feature 02-02

### DISCUSSION THREAD

**Discussion by hyansir** [ppy/osu#19690] [Q&A] [answered]
### Type

Game behaviour

### Bug description

when I try to switch the in game tablet area to full and adjust the sensitivity the sensitivity isn't applied and when i try to turn off the in game tablet area and try to move my cursor with my pen it moves for a bit but stops in between and it repeats
This issue didn't occur when I first installed osu lazer

### Screenshots or videos

![osu_2022-08-10_21-33-24](https://user-images.githubusercontent.com/110999812/183959608-145ed9ec-dc68-4b3b-9cba-8eab95deaeb4.jpg)

https://user-images.githubusercontent.com/110999812/183961524-1293d0d7-002f-45d2-9a14-0f19bba4777d.mp4

### Version

2022.810.2

### Logs

[performance.log](https://github.com/ppy/osu/files/9302100/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9302101/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9302102/updater.log)
[database.log](https://github.com/ppy/osu/files/9302104/database.log)
[network.log](https://github.com/ppy/osu/files/9302106/network.log)

**Comment by peppy** [comment, marked as answer]
You're probably going to have to report this over at https://github.com/OpenTabletDriver/OpenTabletDriver.

**Reply by hyansir** [reply]
actually the otd and wacom drivres were colliding i downloaded a another driver and it solved the issue

**Reply by florbotheflorpus** [reply]
> actually the otd and wacom drivres were colliding i downloaded a another driver and it solved the issue

Do you remember exactly what you did to fix this? I'm having the same problem.

**Reply by hyansir** [reply]
> > actually the otd and wacom drivres were colliding i downloaded a another driver and it solved the issue
> 
> Do you remember exactly what you did to fix this? I'm having the same problem.

so basically what you have to do is if you use wacom drivers the otd drivers will collide with it just download any other tablet driver and it should do the trick ya

**Reply by hyansir** [reply]
or just download open tablet drivers
